src/markdown-convert.ts

```diff
@@ -181,6 +181,7 @@ export async function markdownConvert(
     filesCache,
     mathInlineDelimiters,
     mathBlockDelimiters,
+    mathRenderingOnLineService,
```

Comment:
Hi @Kariiboo , thanks for your pull request. 
Just one question:
I am just wondering maybe `mathRenderingOnlineService` is a better name than `mathRenderingOnLineService`. What do you think?
Thanks again!

Reply:
Actually I think it's fine. I will just merge this PR. Thanks!